src/bin.ts

```diff
@@ -315,7 +315,7 @@ const deploy = async ({ yes, bucket }: { yes: boolean; bucket: string }) => {
         );
 
         await streamToPromise(stream as Readable);
-        await promisifiedParallelLimit(uploadQueue, 20);
+        await promisifiedParallelLimit(uploadQueue, config.parallelLimit ? config.parallelLimit : 20);
```

Comment:
Instead of using a ternary here, could you add the default to DEFAULT_OPTIONS within constants.ts?

Reply:
We considered using DEFAULT_OPTIONS however we saw that previous config params used this pattern e.g. `config.acl` in `bin.ts` on line 235. Would you prefer us to include this in DEFAULT_OPTIONS instead?

Reply:
The reason for config.acl using that pattern is so we can distinguish between `undefined` and `null`. I would prefer DEFAULT_OPTIONS be used in this case, thanks.

Reply:
Made the suggested change to setting the default value. Does this look ok?